package NShardedCache;

message TQueryDocLruCacheConfig {
    // Max number of cached query keys
    optional uint64 MaxQueryCacheSize = 1 [default = 16384];
    // Max documents per cached query key
    optional uint64 MaxDocumentCacheSize = 2 [default = 256];
}

message TDocLruCacheConfig {
    // Max number of cache entries
    optional uint64 MaxCacheSize = 1 [default = 16384];
}
